schools.md

```diff
@@ -702,6 +704,7 @@ Spotswood High School
 Spring Arbor University
 Sreenidhi Institute of Science & Technology
 Sri Sivasubramaniya Nadar College of Engineering
+Stadslyceum 
```

Comment:
The official name of the school may be "The Stadslyceum".  Tough to tell from the website